src/main/java/org/jboss/forge/arquillian/api/YamlGenerator.java

```diff
@@ -23,4 +25,13 @@ public static String toYaml(Map<String, String> params) {
             StringBuilder::toString));
     }
 
+    public static String getYaml(Map<String, Object> params) {
```

Comment:
Actually we have method with def `toYaml` already in this class. Let make it toYml.